### ISSUE TITLE
Crop drag and resize not smooth

### ISSUE THREAD

**Issue by caroig** [Agontuk/vue-cropperjs#102]
I'm using the vue cropper in a large project with vue 2.x, and vuetify.  It all works wonderfully, but the crop window drag and resize has a lot of lag and stutter.  The preview window responds immediately when the crop window is dragged or resized, but the crop window lags behind.  

https://user-images.githubusercontent.com/61102/118612044-afbc7e80-b7bd-11eb-9964-b3ee5e8d97cb.mp4

**Comment by Agontuk** [comment]
Are you handling the drag events ? If so, maybe there's some expensive operation that's causing the lag.
Also check if ui is re-rendering for each drag event.

**Comment by kaushikmakholiyanls** [comment]
Yes same its lagging and also sometime browser hang and crashed too.

**Comment by lateek35** [comment]
@caroig I think that part of your lags is caused by a css transition set on your cropper-crop-box.
